s/sdl2/stone.yaml

```diff
@@ -14,23 +14,16 @@ description : |
     Simple DirectMedia Library
 license     :
     - Zlib
-builddeps   :
-    - pkgconfig(alsa)
-    - pkgconfig(dbus-1)
-    - pkgconfig(egl)
-    - pkgconfig(ibus-1.0)
-    - pkgconfig(libdecor-0)
-    - pkgconfig(libdrm)
-    - pkgconfig(libpipewire-0.3)
-    - pkgconfig(libpulse)
-    - pkgconfig(sndio)
-    - pkgconfig(wayland-client)
-    - pkgconfig(x11)
-    - pkgconfig(xext)
-    - pkgconfig(xkbcommon)
-setup       : |
-    %cmake
-build       : |
-    %cmake_build
 install     : |
-    %cmake_install
+    %install_dir %(installroot)/%(datadir)/tmp
+    touch %(installroot)/%(datadir)/tmp/%(name){,-devel,-dbginfo}
```

Comment:
this doesnt seem consistent with other renames like uutils-coreutils.. ?

Reply:
Should be better now? I'd note that the previous way is how I did it in GCC